src/Containers/CircleTable2.tsx

```diff
@@ -174,7 +208,11 @@ const ChairWrapper = styled.div<IChairWrapper>`
     width: var(--d);
     height: var(--d);
     --az: calc(
-        ${({ counter }) => counter}*1turn / ${({ numOfChairs }) => numOfChairs}
+        ${({ counter, position, numOfChairs }) =>
+                numOfChairs < 3
+                    ? getPositionValue(position)
+                    : counter * 1}turn /
+            ${({ numOfChairs }) => (numOfChairs < 3 ? 1 : numOfChairs)}
```

Comment:
There is quite a bit of logic why not put it all into a function?
Also if you are doing this much logic, chances are your doing something wrong